### ISSUE TITLE
The rules with `::before` and `::after` are detected as invalid rules

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/FiltersCompiler#33]
But these rules are working. Don't remove them.

**Comment by Alex-302** [comment]
```
[17:46:11][ERROR]:Invalid selector: sleazyneasy.com##.video-holder > .video-options::after
[17:46:11][ERROR]:Invalid selector: sleazyneasy.com##.wrap > .sidebar::after
[17:46:24][ERROR]:Invalid selector: thenewslens.com##.social-btns > ul::before
[17:46:24][ERROR]:Invalid selector: news.hankyung.com##.article_wrap > .date + .btn_wrap::after
[17:46:40][ERROR]:Invalid selector: computerbild.de##.paragraph > .labeling::before
[17:46:58][ERROR]:Invalid selector: uol.com.br##.label-publicidade::before
```

**Comment by Alex-302** [comment]
*ping*

**Comment by Alex-302** [comment]
@Mizzick any news?

**Comment by Alex-302** [comment]
```
Invalid selector: northumberlandgazette.co.uk##div[class^="sc-"]::before
Invalid selector: sleazyneasy.com##.video-holder > .video-options::after
```
But these rules are not removed from filter(Base filter)

**Comment by Alex-302** [comment]
No actual anymore.
